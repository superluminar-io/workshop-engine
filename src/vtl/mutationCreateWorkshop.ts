export const request = `
  $util.qr($ctx.stash.put("clerkUsers", $ctx.arguments.input.attendees))
  {}
`;

export const response = `
  $util.toJson($context.result)
`;
