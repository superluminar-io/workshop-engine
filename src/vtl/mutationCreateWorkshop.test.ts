import Parser from 'appsync-template-tester';
import { request, response } from './mutationCreateWorkshop';

describe('request mapping', () => {
  it('should match snapshot', () => {
    const parser = new Parser(request);

    expect(parser.resolve({})).toMatchSnapshot();
  });
});

describe('response mapping', () => {
  it('should match snapshot', () => {
    const parser = new Parser(response);

    const context = {
      result: {
        any: 'data',
      },
    };

    expect(parser.resolve(context)).toMatchSnapshot();
  });
});
