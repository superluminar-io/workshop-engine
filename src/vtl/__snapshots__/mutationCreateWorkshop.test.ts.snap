// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`request mapping should match snapshot 1`] = `Object {}`;

exports[`response mapping should match snapshot 1`] = `
Object {
  "any": "data",
}
`;
